schemas/maas-backend/auth/auth-sms-login/request.json

```diff
@@ -8,7 +8,7 @@
       "type": "object",
```

Comment:
It's probably not in scope of this PR, but as `maas-backend` is our internal project. Shouldn't schemas specific to it be moved into `maas-backend` repository, and here just keep schemas related to TSP adapters?

Reply:
@medikoo @konker and i had the same discussion, not yet came to a conclusion. If we move `maas-backend` specific schemas to `maas-backend` repo then we need to first figure out a way to de-ref the schemas. The module we're using now support only relative path in $ref tag, so what we want is not yet possible